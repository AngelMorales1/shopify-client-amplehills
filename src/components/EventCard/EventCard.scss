@import 'settings', 'typography', 'utils';

.EventCard {
  @extend .transition;
  display: none;
  opacity: 0;
  border-radius: radius('s');
  min-height: spacing(144);
  max-width: spacing(86);
  flex-direction: column;

  @media ($medium-up) {
    max-width: inherit;
    min-width: spacing(200);
    min-height: spacing(96);
    flex-direction: row;
  }

  &--active {
    display: flex;
    opacity: 1;
  }

  &__image {
    border-top-left-radius: radius('s');
    border-top-right-radius: radius('s');
    height: spacing(72);

    @media ($medium-up) {
      border-radius: 0;
      border-top-left-radius: radius('s');
      border-bottom-left-radius: radius('s');
      height: inherit;
      flex: auto;
    }
  }

  &__text-container {
    padding: spacing(8);
    flex: auto;

    @media ($medium-up) {
      padding: spacing(12);
    }
  }

  &__date-text {
    font-size: $font-2;
    letter-spacing: spacing(0.2);
    line-height: $font-6;
    color: color('peach');
    font-weight: 800;

    @media ($medium-up) {
      font-size: $font-3;
      color: color('madison-blue');
      font-weight: inherit;
    }
  }

  &__text {
    @extend .tout;
    line-height: $font-6;

    > strong {
      @extend .bold;
    }
  }

  &__text-container {
    padding: spacing(8);
    height: spacing(72);

    @media ($medium-up) {
      padding: spacing(12);
      height: 100%;
    }
  }

  &__text {
    @extend .tout;
    line-height: $font-6;
  }

  &__title {
    font-family: $font-carter;
    font-size: $font-5;

    @media ($medium-up) {
      font-size: $font-6;
    }
  }

  &__button {
    display: inline-flex;
  }
}
