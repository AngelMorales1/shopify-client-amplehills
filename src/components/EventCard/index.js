import React from 'react';
import PropTypes from 'prop-types';
import get from 'utils/get';
import contentfulImgUtil from 'utils/contentfulImgUtil';
import cx from 'classnames';
import moment from 'moment';

import styles from './EventCard.scss';
import { Image, Button } from 'components/base';

const EventCard = ({ event }) => {
  console.log(event);
  const fields = get(event, 'fields', {});
  const image = get(fields, 'image.fields.file.url', '');
  const title = get(fields, 'title', '');
  const date = moment(get(fields, 'date', '')).format('MMMM D YYYY');
  const time = get(fields, 'time', '');
  const location = get(fields, 'location.fields.title', '');

  return (
    <div
      className={cx(styles['EventCard'], 'bg-white mb3 flex col-12 md-col-7')}
    >
      <div
        className={cx(styles['EventCard__image'], 'col-12 md-col-6')}
        style={{
          background: `url(${contentfulImgUtil(
            image,
            '900'
          )}) no-repeat center`,
          backgroundSize: 'cover'
        }}
      />
      <div className="flex flex-column justify-between col-12 md-col-6 p4">
        <div>
          <span className="tout">{`${date}, ${time} at ${location}`}</span>
          <h2 className={cx(styles['EventCard__title'], 'mt2')}>{title}</h2>
        </div>
        <Button color="peach" label="RSVP" />
      </div>
    </div>
  );
};

EventCard.propTypes = {};

EventCard.defaultProps = {};

export default EventCard;
