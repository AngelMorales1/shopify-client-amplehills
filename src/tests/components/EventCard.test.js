import React from 'react';
import { shallow } from 'enzyme';

import EventCard from 'components/EventCard';

it('renders without data', () => {
  const component = shallow(<EventCard />);

  expect(component).toMatchSnapshot();
});

it('renders with data', () => {
  const component = shallow(
    <EventCard
      event={{
        id: 'foo',
        blockCardText: 'foo',
        eventType: 'foo',
        image: 'foo',
        locationTitle: 'foo',
        title: 'foo',
        datesAndTimes: [
          {
            uuid: 'foo',
            Date: 'foo',
            Time: 'foo'
          }
        ]
      }}
      active={false}
    />
  );

  expect(component).toMatchSnapshot();
});
