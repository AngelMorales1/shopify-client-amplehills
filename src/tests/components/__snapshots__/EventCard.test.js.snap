// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with data 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <EventCard
    active={false}
    event={
        Object {
            "blockCardText": "foo",
            "datesAndTimes": Array [
              Object {
                "Date": "foo",
                "Time": "foo",
                "uuid": "foo",
              },
            ],
            "eventType": "foo",
            "id": "foo",
            "image": "foo",
            "locationTitle": "foo",
            "title": "foo",
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="col-12 md-col-6"
          style={
                    Object {
                              "background": "url(foo?w=900&fm=jpg&fl=progressive&q=90) no-repeat center",
                              "backgroundSize": "cover",
                            }
          }
/>,
        <div
          className="col-12 md-col-6 flex flex-column justify-between"
>
          <div>
                    <p
                              className="mb2"
                    >
<<<<<<< HEAD
                              Invalid date, undefined at foo
=======
                              Invalid date, foo: at foo
>>>>>>> Add EventDetailHero text
                    </p>
                    <h2
                              className=""
                    >
                              foo
                    </h2>
                    <p
                              className="mt1 mb3"
                              dangerouslySetInnerHTML={
                                        Object {
                                                  "__html": "<p>foo</p>
                                        ",
                                                }
                              }
                    />
          </div>
          <div>
                    <Button
                              className="inline-flex"
                              color="peach"
                              disabled={false}
                              fullWidth={false}
                              hover="shadow"
                              label="RSVP"
                              onClick={[Function]}
                              shadow={false}
<<<<<<< HEAD
                              to="/events/undefined"
=======
                              to="/events/foo"
>>>>>>> Add EventDetailHero text
                              type="button"
                              variant="primary"
                    />
          </div>
</div>,
      ],
      "className": "transition-slide-up-large bg-white mb3 flex col-12 md-col-7",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "col-12 md-col-6",
          "style": Object {
            "background": "url(foo?w=900&fm=jpg&fl=progressive&q=90) no-repeat center",
            "backgroundSize": "cover",
          },
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div>
              <p
                            className="mb2"
              >
<<<<<<< HEAD
                            Invalid date, undefined at foo
=======
                            Invalid date, foo: at foo
>>>>>>> Add EventDetailHero text
              </p>
              <h2
                            className=""
              >
                            foo
              </h2>
              <p
                            className="mt1 mb3"
                            dangerouslySetInnerHTML={
                                          Object {
                                                        "__html": "<p>foo</p>
                                          ",
                                                      }
                            }
              />
</div>,
            <div>
              <Button
                            className="inline-flex"
                            color="peach"
                            disabled={false}
                            fullWidth={false}
                            hover="shadow"
                            label="RSVP"
                            onClick={[Function]}
                            shadow={false}
<<<<<<< HEAD
                            to="/events/undefined"
=======
                            to="/events/foo"
>>>>>>> Add EventDetailHero text
                            type="button"
                            variant="primary"
              />
</div>,
          ],
          "className": "col-12 md-col-6 flex flex-column justify-between",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <p
                  className="mb2"
>
<<<<<<< HEAD
                  Invalid date, undefined at foo
=======
                  Invalid date, foo: at foo
>>>>>>> Add EventDetailHero text
</p>,
                <h2
                  className=""
>
                  foo
</h2>,
                null,
                <p
                  className="mt1 mb3"
                  dangerouslySetInnerHTML={
                                    Object {
                                                      "__html": "<p>foo</p>
                                    ",
                                                    }
                  }
/>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
<<<<<<< HEAD
                  "children": "Invalid date, undefined at foo",
                  "className": "mb2",
                },
                "ref": null,
                "rendered": "Invalid date, undefined at foo",
=======
                  "children": "Invalid date, foo: at foo",
                  "className": "mb2",
                },
                "ref": null,
                "rendered": "Invalid date, foo: at foo",
>>>>>>> Add EventDetailHero text
                "type": "p",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "foo",
                  "className": "",
                },
                "ref": null,
                "rendered": "foo",
                "type": "h2",
              },
              null,
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "mt1 mb3",
                  "dangerouslySetInnerHTML": Object {
                    "__html": "<p>foo</p>
",
                  },
                },
                "ref": null,
                "rendered": null,
                "type": "p",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Button
                className="inline-flex"
                color="peach"
                disabled={false}
                fullWidth={false}
                hover="shadow"
                label="RSVP"
                onClick={[Function]}
                shadow={false}
<<<<<<< HEAD
                to="/events/undefined"
=======
                to="/events/foo"
>>>>>>> Add EventDetailHero text
                type="button"
                variant="primary"
/>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": null,
                "className": "inline-flex",
                "color": "peach",
                "disabled": false,
                "fullWidth": false,
                "hover": "shadow",
                "label": "RSVP",
                "onClick": [Function],
                "shadow": false,
<<<<<<< HEAD
                "to": "/events/undefined",
=======
                "to": "/events/foo",
>>>>>>> Add EventDetailHero text
                "type": "button",
                "variant": "primary",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="col-12 md-col-6"
            style={
                        Object {
                                    "background": "url(foo?w=900&fm=jpg&fl=progressive&q=90) no-repeat center",
                                    "backgroundSize": "cover",
                                  }
            }
/>,
          <div
            className="col-12 md-col-6 flex flex-column justify-between"
>
            <div>
                        <p
                                    className="mb2"
                        >
<<<<<<< HEAD
                                    Invalid date, undefined at foo
=======
                                    Invalid date, foo: at foo
>>>>>>> Add EventDetailHero text
                        </p>
                        <h2
                                    className=""
                        >
                                    foo
                        </h2>
                        <p
                                    className="mt1 mb3"
                                    dangerouslySetInnerHTML={
                                                Object {
                                                            "__html": "<p>foo</p>
                                                ",
                                                          }
                                    }
                        />
            </div>
            <div>
                        <Button
                                    className="inline-flex"
                                    color="peach"
                                    disabled={false}
                                    fullWidth={false}
                                    hover="shadow"
                                    label="RSVP"
                                    onClick={[Function]}
                                    shadow={false}
<<<<<<< HEAD
                                    to="/events/undefined"
=======
                                    to="/events/foo"
>>>>>>> Add EventDetailHero text
                                    type="button"
                                    variant="primary"
                        />
            </div>
</div>,
        ],
        "className": "transition-slide-up-large bg-white mb3 flex col-12 md-col-7",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "col-12 md-col-6",
            "style": Object {
              "background": "url(foo?w=900&fm=jpg&fl=progressive&q=90) no-repeat center",
              "backgroundSize": "cover",
            },
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div>
                <p
                                className="mb2"
                >
<<<<<<< HEAD
                                Invalid date, undefined at foo
=======
                                Invalid date, foo: at foo
>>>>>>> Add EventDetailHero text
                </p>
                <h2
                                className=""
                >
                                foo
                </h2>
                <p
                                className="mt1 mb3"
                                dangerouslySetInnerHTML={
                                                Object {
                                                                "__html": "<p>foo</p>
                                                ",
                                                              }
                                }
                />
</div>,
              <div>
                <Button
                                className="inline-flex"
                                color="peach"
                                disabled={false}
                                fullWidth={false}
                                hover="shadow"
                                label="RSVP"
                                onClick={[Function]}
                                shadow={false}
<<<<<<< HEAD
                                to="/events/undefined"
=======
                                to="/events/foo"
>>>>>>> Add EventDetailHero text
                                type="button"
                                variant="primary"
                />
</div>,
            ],
            "className": "col-12 md-col-6 flex flex-column justify-between",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <p
                    className="mb2"
>
<<<<<<< HEAD
                    Invalid date, undefined at foo
=======
                    Invalid date, foo: at foo
>>>>>>> Add EventDetailHero text
</p>,
                  <h2
                    className=""
>
                    foo
</h2>,
                  null,
                  <p
                    className="mt1 mb3"
                    dangerouslySetInnerHTML={
                                        Object {
                                                            "__html": "<p>foo</p>
                                        ",
                                                          }
                    }
/>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
<<<<<<< HEAD
                    "children": "Invalid date, undefined at foo",
                    "className": "mb2",
                  },
                  "ref": null,
                  "rendered": "Invalid date, undefined at foo",
=======
                    "children": "Invalid date, foo: at foo",
                    "className": "mb2",
                  },
                  "ref": null,
                  "rendered": "Invalid date, foo: at foo",
>>>>>>> Add EventDetailHero text
                  "type": "p",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "foo",
                    "className": "",
                  },
                  "ref": null,
                  "rendered": "foo",
                  "type": "h2",
                },
                null,
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "mt1 mb3",
                    "dangerouslySetInnerHTML": Object {
                      "__html": "<p>foo</p>
",
                    },
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "p",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <Button
                  className="inline-flex"
                  color="peach"
                  disabled={false}
                  fullWidth={false}
                  hover="shadow"
                  label="RSVP"
                  onClick={[Function]}
                  shadow={false}
<<<<<<< HEAD
                  to="/events/undefined"
=======
                  to="/events/foo"
>>>>>>> Add EventDetailHero text
                  type="button"
                  variant="primary"
/>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "children": null,
                  "className": "inline-flex",
                  "color": "peach",
                  "disabled": false,
                  "fullWidth": false,
                  "hover": "shadow",
                  "label": "RSVP",
                  "onClick": [Function],
                  "shadow": false,
<<<<<<< HEAD
                  "to": "/events/undefined",
=======
                  "to": "/events/foo",
>>>>>>> Add EventDetailHero text
                  "type": "button",
                  "variant": "primary",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`renders without data 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <EventCard
    active={false}
    event={
        Object {
            "blockCardText": "",
            "datesAndTimes": Array [
              Object {
                "Date": "",
                "Time": "",
                "uuid": "",
              },
            ],
            "eventType": "",
            "id": "",
            "image": "",
            "locationTitle": "",
            "title": "",
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="col-12 md-col-6"
          style={
                    Object {
                              "background": "url(?w=900&fm=jpg&fl=progressive&q=90) no-repeat center",
                              "backgroundSize": "cover",
                            }
          }
/>,
        <div
          className="col-12 md-col-6 flex flex-column justify-between"
>
          <div>
                    <p
                              className="mb2"
                    >
                              Invalid date, undefined at 
                    </p>
                    <h2
                              className=""
                    />
          </div>
          <div>
                    <Button
                              className="inline-flex"
                              color="peach"
                              disabled={false}
                              fullWidth={false}
                              hover="shadow"
                              label="RSVP"
                              onClick={[Function]}
                              shadow={false}
<<<<<<< HEAD
                              to="/events/undefined"
=======
                              to="/events/"
>>>>>>> Add EventDetailHero text
                              type="button"
                              variant="primary"
                    />
          </div>
</div>,
      ],
      "className": "transition-slide-up-large bg-white mb3 flex col-12 md-col-7",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "col-12 md-col-6",
          "style": Object {
            "background": "url(?w=900&fm=jpg&fl=progressive&q=90) no-repeat center",
            "backgroundSize": "cover",
          },
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div>
              <p
                            className="mb2"
              >
                            Invalid date, undefined at 
              </p>
              <h2
                            className=""
              />
</div>,
            <div>
              <Button
                            className="inline-flex"
                            color="peach"
                            disabled={false}
                            fullWidth={false}
                            hover="shadow"
                            label="RSVP"
                            onClick={[Function]}
                            shadow={false}
<<<<<<< HEAD
                            to="/events/undefined"
=======
                            to="/events/"
>>>>>>> Add EventDetailHero text
                            type="button"
                            variant="primary"
              />
</div>,
          ],
          "className": "col-12 md-col-6 flex flex-column justify-between",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <p
                  className="mb2"
>
                  Invalid date, undefined at 
</p>,
                <h2
                  className=""
/>,
                null,
                null,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Invalid date, undefined at ",
                  "className": "mb2",
                },
                "ref": null,
                "rendered": "Invalid date, undefined at ",
                "type": "p",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "",
                  "className": "",
                },
                "ref": null,
                "rendered": "",
                "type": "h2",
              },
              null,
              null,
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Button
                className="inline-flex"
                color="peach"
                disabled={false}
                fullWidth={false}
                hover="shadow"
                label="RSVP"
                onClick={[Function]}
                shadow={false}
<<<<<<< HEAD
                to="/events/undefined"
=======
                to="/events/"
>>>>>>> Add EventDetailHero text
                type="button"
                variant="primary"
/>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": null,
                "className": "inline-flex",
                "color": "peach",
                "disabled": false,
                "fullWidth": false,
                "hover": "shadow",
                "label": "RSVP",
                "onClick": [Function],
                "shadow": false,
<<<<<<< HEAD
                "to": "/events/undefined",
=======
                "to": "/events/",
>>>>>>> Add EventDetailHero text
                "type": "button",
                "variant": "primary",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="col-12 md-col-6"
            style={
                        Object {
                                    "background": "url(?w=900&fm=jpg&fl=progressive&q=90) no-repeat center",
                                    "backgroundSize": "cover",
                                  }
            }
/>,
          <div
            className="col-12 md-col-6 flex flex-column justify-between"
>
            <div>
                        <p
                                    className="mb2"
                        >
                                    Invalid date, undefined at 
                        </p>
                        <h2
                                    className=""
                        />
            </div>
            <div>
                        <Button
                                    className="inline-flex"
                                    color="peach"
                                    disabled={false}
                                    fullWidth={false}
                                    hover="shadow"
                                    label="RSVP"
                                    onClick={[Function]}
                                    shadow={false}
<<<<<<< HEAD
                                    to="/events/undefined"
=======
                                    to="/events/"
>>>>>>> Add EventDetailHero text
                                    type="button"
                                    variant="primary"
                        />
            </div>
</div>,
        ],
        "className": "transition-slide-up-large bg-white mb3 flex col-12 md-col-7",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "col-12 md-col-6",
            "style": Object {
              "background": "url(?w=900&fm=jpg&fl=progressive&q=90) no-repeat center",
              "backgroundSize": "cover",
            },
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div>
                <p
                                className="mb2"
                >
                                Invalid date, undefined at 
                </p>
                <h2
                                className=""
                />
</div>,
              <div>
                <Button
                                className="inline-flex"
                                color="peach"
                                disabled={false}
                                fullWidth={false}
                                hover="shadow"
                                label="RSVP"
                                onClick={[Function]}
                                shadow={false}
<<<<<<< HEAD
                                to="/events/undefined"
=======
                                to="/events/"
>>>>>>> Add EventDetailHero text
                                type="button"
                                variant="primary"
                />
</div>,
            ],
            "className": "col-12 md-col-6 flex flex-column justify-between",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <p
                    className="mb2"
>
                    Invalid date, undefined at 
</p>,
                  <h2
                    className=""
/>,
                  null,
                  null,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Invalid date, undefined at ",
                    "className": "mb2",
                  },
                  "ref": null,
                  "rendered": "Invalid date, undefined at ",
                  "type": "p",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "",
                    "className": "",
                  },
                  "ref": null,
                  "rendered": "",
                  "type": "h2",
                },
                null,
                null,
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <Button
                  className="inline-flex"
                  color="peach"
                  disabled={false}
                  fullWidth={false}
                  hover="shadow"
                  label="RSVP"
                  onClick={[Function]}
                  shadow={false}
<<<<<<< HEAD
                  to="/events/undefined"
=======
                  to="/events/"
>>>>>>> Add EventDetailHero text
                  type="button"
                  variant="primary"
/>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "children": null,
                  "className": "inline-flex",
                  "color": "peach",
                  "disabled": false,
                  "fullWidth": false,
                  "hover": "shadow",
                  "label": "RSVP",
                  "onClick": [Function],
                  "shadow": false,
<<<<<<< HEAD
                  "to": "/events/undefined",
=======
                  "to": "/events/",
>>>>>>> Add EventDetailHero text
                  "type": "button",
                  "variant": "primary",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
